modules/pipeline/pipengine/actionexecutor/plugins/k8sspark/k8sspark.go

```diff
@@ -45,16 +47,40 @@ const (
 )
 
 func init() {
-	types.MustRegister(Kind, func(name types.Name, clusterName string, cluster apistructs.ClusterInfo) (types.TaskExecutor, error) {
-		k, err := New(name, clusterName, cluster)
+	types.MustRegister(Kind, func(name types.Name, options map[string]string) (types.ActionExecutor, error) {
+		clusterName, err := Kind.GetClusterNameByExecutorName(name)
+		if err != nil {
+			return nil, err
+		}
+		cluster, err := clusterinfo.GetClusterByName(clusterName)
 		if err != nil {
 			return nil, err
 		}
-		return k, nil
+		return New(name, clusterName, cluster)
 	})
 }
 
-func (k *K8sSpark) Create(ctx context.Context, task *spec.PipelineTask) (interface{}, error) {
+func (k *K8sSpark) Start(ctx context.Context, task *spec.PipelineTask) (data interface{}, err error) {
+	defer k.errWrapper.WrapTaskError(&err, "start job", task)
+	if err := logic.ValidateAction(task); err != nil {
+		return nil, err
+	}
+	created, started, err := k.Exist(ctx, task)
+	if err != nil {
+		return nil, err
+	}
+	if !created {
+		logrus.Warnf("%s: task not created, try to create actionInfo: %s", k.Kind().String(), logic.PrintTaskInfo(task))
```

Comment:
what is `try to create actionInfo`?

Reply:
adjusted, k8sjob, k8sflink k8sspark had change to `%s: task not created(auto try to create), taskInfo: %s`